views/consulting.html

```diff
@@ -69,8 +69,8 @@ <h3>Adam</h3>
                   <img src="static/img/irakli.jpg" class="center-block"/>
               </div>
               <div class="col-sm-9">
-                <h3>Irakali</h3>
-                <p>Irakali is a Self-educated Python and JavaScript developer with over 2 years of experience of working with data-driven projects and applications on the back-end.
+                <h3>Irakli</h3>
+                <p>Irakli is a Self-educated Python and JavaScript developer with over 2 years of experience of working with data-driven projects and applications on the back-end.
```

Comment:
@tanvirchahal  This is correct but why "Self-educated" is capitalized?

Reply:
@anuveyatsu passed it via proofreader, didn't raise an alarm. Shall i lowercase that?